presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -75,6 +75,58 @@ constexpr char const* kTaskUriFormat =
     "{}://{}:{}"; // protocol, address and port
 constexpr char const* kConnectorName = "connector.name";
 
+// Set of valid configs for the hive connector.
+const std::unordered_set<std::string> kHiveConfigs = {
```

Comment:
This file keeps growing ...
I almost think that the Connectors should be refactored a bit - i.e. move `class HivePrestoToVeloxConnector final : public PrestoToVeloxConnector` to their own file.
And there this list and validation can be added. In this case everything is together - the config for the hive connector. 
Or something that logically combines them as applicable together. Just adding them here is a bit too specific.

@majetideepak What do you think?

Reply:
Thanks for the suggestion, updated accordingly.